js/languages/en-US.json

```diff
@@ -1020,6 +1027,10 @@
     "addQueryProviderBtn": "Add search provider",
     "chooseDefaultTitle": "Select a default search provider",
     "chooseDefaultMsg": "This provider will be used as the default for future seaches until you set a new one.",
+    "providers": {
+        "transactions": "Transactions",
+        "myPage": "My Page",
```

Comment:
In json, you can't have a comma on the last item in a set. This comma will prevent the app from starting.

Always check to make sure nothing breaks after editing any file.